### PR TITLE
docs: C# client usage - II

### PR DESCRIPTION
**Pull request by Anush008**
Part-2 of #599.

Adds the C# client usage to the guides and remaining sections of the Qdrant docs.

[Preview](https://deploy-preview-607--condescending-goldwasser-91acf0.netlify.app/documentation/guides/multiple-partitions/)

### CODE REVIEW

**Comment by Anush008** [comment]
@IvanPleshkov, named args here as well.
https://github.com/qdrant/landing_page/pull/607/commits/ed8835694fb3dd1f07cefab0459f4f396f6513d3

File: qdrant-landing/content/documentation/guides/optimize.md [hunk 1]
```diff
@@ -259,6 +276,23 @@ client
...
 
+```csharp
+using Qdrant.Client;
+using Qdrant.Client.Grpc;
+
+var client = new QdrantClient("localhost", 6334);
+
+await client.SearchAsync(
+	"{collection_name}",
+	new float[] { 0.2f, 0.1f, 0.9f, 0.7f },
```
Comment (IvanPleshkov) [inline qdrant-landing/content/documentation/guides/optimize.md:287]: I propose to add names to other arguments

File: qdrant-landing/content/documentation/guides/optimize.md [hunk 2]
```diff
@@ -845,3 +937,15 @@ client
...
 ```
 
+```csharp
+using Qdrant.Client;
+using Qdrant.Client.Grpc;
+
+var client = new QdrantClient("localhost", 6334);
+
+await client.CreateCollectionAsync(
+	"{collection_name}",
```
Comment (IvanPleshkov) [inline qdrant-landing/content/documentation/guides/optimize.md:947]: I propose to add `vectorsConfig` and `collectionName`

File: qdrant-landing/content/documentation/guides/quantization.md [hunk 3]
```diff
@@ -669,6 +722,28 @@ client
     .get();
 ```
 
+```csharp
+using Qdrant.Client;
+using Qdrant.Client.Grpc;
+
+var client = new QdrantClient("localhost", 6334);
+
+await client.SearchAsync(
```
Comment (IvanPleshkov) [inline qdrant-landing/content/documentation/guides/quantization.md:731]: I propose to add vectorsConfig and collectionName

**Review by IvanPleshkov — APPROVED** [review]
Amazing job

File: qdrant-landing/content/documentation/guides/optimize.md (hunk 1 above)
Comment (Anush008) [reply qdrant-landing/content/documentation/guides/optimize.md]: https://github.com/qdrant/landing_page/pull/607/commits/d14b76c88a046f08c2bd47abef6f98a437c1ec15

**Pull request merged**